IO/Ex1.hpp

```diff
@@ -0,0 +1,24 @@
+
+#include <fstream> //ios_base::openmode
+#include <string>
+
+class FileHandler {
+public:
+    explicit FileHandler(const std::string path, std::ios_base::openmode permissions); //Ctor
+    FileHandler(const FileHandler &other); //CCtor
+    FileHandler(FileHandler &&other) noexcept; //MCtor
+
+    FileHandler& operator=(const FileHandler &other); //Copy Assignment
+    FileHandler& operator=(FileHandler &&other) noexcept; //Move Assignment
+
+    std::fstream &getFD();
+    static void createFile(const std::string filename);
+
+    ~FileHandler() noexcept = default; //Dtor
+
+private:
+    std::fstream m_fd;
```

Comment:
fstreams are not file descriptors

Reply:
do you mean i should replace the variable name ?  (to somthing like m_file_stream ?)